### ISSUE TITLE
1

### ISSUE THREAD

**Issue by doggg79** [hslcode/CV_CNN_DOA#1]

**Comment by hslcode** [comment]
Thank you for your question, there is an upcoming paper of ours that will be published in ICICSP 2023 available for you to reference, but it has not been published yet, I will update the reference link once the paper is published. If this code is helpful to you, please cite our paper as shown in the following:
{
title={Robust DOA Estimation Using Deep Complex-Valued Convolutional Networks with Sparse Prior}, 
author={Hu, Shulin and Zeng, Cao and Liu, Minti and Tao, Haihong and Zhao, Shihua and Liu, Yu},
booktitle={2023 6th International Conference on Information Communication and Signal Processing (ICICSP)}, 
 pages={234--239},
 year={2023},
 organization={IEEE}
}
In addition, you can refer to the following published related works:
[1]	Z. Liu, C. Zhang, and P. S. Yu, “Direction-of-Arrival Estimation Based on Deep Neural Networks With Robustness to Array Imperfections,” IEEE Trans. Antennas Propag., vol. 66, no. 12, pp. 7315–7327, Dec. 2018.
[2]	L. Wu, Z. Liu, and Z. Huang, “Deep convolution network for direction of arrival estimation with sparse prior,” IEEE Signal Process. Lett., vol. 26, no. 11, pp. 1688–1692, Nov. 2019. 
[3]	G. K. Papageorgiou, M. Sellathurai, and Y. C. Eldar, “Deep Networks for Direction-of-Arrival Estimation in Low SNR,” IEEE Trans. Signal Process., vol. 69, pp. 3714–3729, 2021.
[4]	Ma, Y., Zeng, Y., & Sun, S. (2022). A deep learning based super resolution DoA estimator with single snapshot MIMO radar data. IEEE Transactions on Vehicular Technology, 71(4), 4142-4155.

**Comment by qihuixu1** [comment]
Thank you for your reply, I wonder if there are preprints available for early viewing? Also it seems that musicdoa.m and rootmusicdoa.m are missing from the code file.

**Comment by qihuixu1** [comment]
> Thank you for your question, there is an upcoming paper of ours that will be published in ICICSP 2023 available for you to reference, but it has not been published yet, I will update the reference link once the paper is published. If this code is helpful to you, please cite our paper as shown in the following: { title={Robust DOA Estimation Using Deep Complex-Valued Convolutional Networks with Sparse Prior}, author={Hu, Shulin and Zeng, Cao and Liu, Minti and Tao, Haihong and Zhao, Shihua and Liu, Yu}, booktitle={2023 6th International Conference on Information Communication and Signal Processing (ICICSP)}, pages={234--239}, year={2023}, organization={IEEE} } In addition, you can refer to the following published related works: [1] Z. Liu, C. Zhang, and P. S. Yu, “Direction-of-Arrival Estimation Based on Deep Neural Networks With Robustness to Array Imperfections,” IEEE Trans. Antennas Propag., vol. 66, no. 12, pp. 7315–7327, Dec. 2018. [2] L. Wu, Z. Liu, and Z. Huang, “Deep convolution network for direction of arrival estimation with sparse prior,” IEEE Signal Process. Lett., vol. 26, no. 11, pp. 1688–1692, Nov. 2019. [3] G. K. Papageorgiou, M. Sellathurai, and Y. C. Eldar, “Deep Networks for Direction-of-Arrival Estimation in Low SNR,” IEEE Trans. Signal Process., vol. 69, pp. 3714–3729, 2021. [4] Ma, Y., Zeng, Y., & Sun, S. (2022). A deep learning based super resolution DoA estimator with single snapshot MIMO radar data. IEEE Transactions on Vehicular Technology, 71(4), 4142-4155.

Thank you for your reply, I wonder if there are preprints available for early viewing? Also it seems that musicdoa.m and rootmusicdoa.m are missing from the code file.

**Comment by hslcode** [comment]
Of course, we can provide you with a preprint version. Please leave your email.

**Comment by myy378627980** [comment]
> Of course, we can provide you with a preprint version. Please leave your email.

Hello, could you also send me a preprint version to review and learn in advance? My email is 378627980@qq.com. Thank you very much!

**Comment by hslcode** [comment]
> > Of course, we can provide you with a preprint version. Please leave your email.
> 
> Hello, could you also send me a preprint version to review and learn in advance? My email is [378627980@qq.com](mailto:378627980@qq.com). Thank you very much!

Thank you for your interest, an preprint of our paper has been emailed to you, please check it out, thank you.

**Comment by Lhuifang** [comment]
Hello, I 'm very interested in the content of your article.Could you also provide me with a preprint version to learn? My email is 2592094879@qq.com. Thank you very much!

**Comment by hslcode** [comment]
> Hello, I 'm very interested in the content of your article.Could you also provide me with a preprint version to learn? My email is [2592094879@qq.com](mailto:2592094879@qq.com). Thank you very much!

Thanks for your attention, The published paper is available at https://ieeexplore.ieee.org/document/10390873.

**Comment by Lhuifang** [comment]
Hello ! Excuse me again, I want to ask you some questions about the code you provide for learning. How to obtain 'TRAIN _ DATA _ 16ULA _ K2 _ min20to10dB _ res1 _ 60deg _ ECM.h5 ' in the train.py file ? Thank you very much！

**Comment by hslcode** [comment]
> Hello ! Excuse me again, I want to ask you some questions about the code you provide for learning. How to obtain 'TRAIN _ DATA _ 16ULA _ K2 _ min20to10dB _ res1 _ 60deg _ ECM.h5 ' in the train.py file ? Thank you very much！

Hi, you can obtain this data file by running CV_CNN_DOA/Code/Matlab/GENER_Train_Data.m. Thank you.
